tests/gold_tests/pluginTest/lua/metrics.sh

```diff
@@ -0,0 +1,31 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+N=60
+while (( N > 0 ))
+do
+    rm -f metrics.out metrics.txt
+    traffic_ctl metric match lua > metrics.out
```

Comment:
would it make more sense to just get all the metrics by explicitly mention their names?  can occur that we  get more metrics than the ones you defined in the `metrics.gold`? probably *not* as this is isolated in this test(worth asking). In any case if you request for a record that does not exist you will get an error out of the `traffic_ctl` (in case you request  `traffic_ctl metric get a b c`

Reply:
Hmm, I don't know enough about metrics generally to know if there is a risk that other metrics would appear in the output in a pseudo-random fashion.  I did not see that in my testing.